tests/aws/services/lambda_/event_source_mapping/utils.py

```diff
@@ -1,6 +1,57 @@
 from localstack.config import LAMBDA_EVENT_SOURCE_MAPPING
 from localstack.testing.aws.util import is_aws_cloud
 
+# For DynamoDB Streams and Kinesis:
+# If the batchItemFailures array contains multiple items, Lambda uses the record with the lowest sequence number as the checkpoint.
+# Lambda then retries all records starting from that checkpoint.
+LAMBDA_DYNAMODB_BATCH_ITEM_FAILURE = """
```

Comment:
I think creating files for Lambda functions improves readability using syntax highlighting